inspektor/inspector.py

```diff
@@ -77,9 +77,12 @@ def is_python(self):
         return self.is_script(language='python')
 
     def is_toignore(self):
-        for pat in self.ignore:
-            if fnmatch.fnmatch(self.path, pat):
+        path = self.path
+        if path.startswith('./'):
+            path = path[2:]
+        for pattern in self.ignore:
+            if fnmatch.fnmatch(path, pattern):
                 return True
-            if self.path.startswith(os.path.abspath(pat)):
+            if pattern in self.path:
```

Comment:
Be more lenient with the skip patterns, for example::

    inspekt style --exclude .venv

To exclude all .venv related checks, while allowing the alternate form::

    inspekt style --exclude .venv*

So yeah, basically be less picky about the exclude patterns. Maybe not a good idea indeed.